sunpycube/cube/datacube.py

```diff
@@ -495,11 +496,32 @@ def _array_is_aligned(self):
     def __getitem__(self, item):
         if item is None or (isinstance(item, tuple) and None in item):
             raise IndexError("None indices not supported")
-        if not self._array_is_aligned():
-            raise cu.CubeError(6, "Slicing on unaligned wcs-array systems " +
-                               "not supported at the moment")
```

Comment:
it was failing here and you said something about physical coordinated @DanRyanIrish